src/device/r4300/cp1.h

```diff
@@ -25,6 +25,11 @@
 #include <stdint.h>
 #include "new_dynarec/new_dynarec.h"
 
+#if defined(__SSE__) || defined(_M_X64) || (_M_IX86_FP > 0)
+#include <immintrin.h>
+#define SSE
```

Comment:
I think we need a better name than simply `SSE` that could clash with other constants and could be easier to search (as a reference to this particular problem). `M64P_SSE_NO_FTZ`?

Reply:
I would maybe change it to SSE_CAPABLE or something, the flag is not specific to FTZ or anything, it's just to determine whether SSE is supported or not.

Reply:
I get the idea, but I don't think it's a valid point. My problem is about the meaning :+1: 

Having a `SSE_CAPABLE` in the middle of the code (device/r4300/cp1.h) doesn't explain what is the purpose of the variable.

As the reason of the check is to fix a FTZ problem, I would keep this information on the variable name.

Reply:
The purpose of the variable is just to determine whether SSE is supported or not, I guess that is where we see it different? It's unrelated to FTZ/DAZ

Reply:
You are partially correct. The purpose on the variable is to determine whether SSE is supported or not, but because of the placement (globally, in a particular component, under an include intrinsic), its purpose must be specific. That's why I was suggesting a specific name.

Can I suggest to put this in `osal.h` as `OSAL_SSE` with OS specific check ?

Reply:
Sure, I've updated it to use osal